code/modules/mob/living/carbon/human/human.dm

```diff
@@ -835,43 +835,75 @@
+		if(!src.is_busy && src.zone_selected == BODY_ZONE_HEAD || src.zone_selected == BODY_ZONE_PRECISE_GROIN && get_turf(src) == get_turf(T) && !(T.mobility_flags & MOBILITY_STAND) && src.a_intent != INTENT_HELP) //all the stars align, time to curbstomp
 			src.is_busy = TRUE
 
-			if (!do_mob(src,T,25) || src.zone_selected != BODY_ZONE_HEAD || get_turf(src) != get_turf(T) || (T.mobility_flags & MOBILITY_STAND) || src.a_intent == INTENT_HELP) //wait 30ds and make sure the stars still align
+			if (!do_mob(src,T,25) || get_turf(src) != get_turf(T) || (T.mobility_flags & MOBILITY_STAND) || src.a_intent == INTENT_HELP) //wait 30ds and make sure the stars still align (Body zone check removed after PR #958)
 				src.is_busy = FALSE
 				return
 
 			T.Stun(6)
 
-			var/increment = (T.lying/90)-2
-			setDir(increment > 0 ? WEST : EAST)
-			for(var/i in 1 to 5)
-				src.pixel_y += 8-i
-				src.pixel_x -= increment
-				sleep(0.2)
-			for(var/i in 1 to 5)
-				src.pixel_y -= 8-i
-				src.pixel_x -= increment
-				sleep(0.2)
-
-			playsound(src, 'sound/effects/hit_kick.ogg', 80, 1, -1)
-			playsound(src, 'sound/weapons/punch2.ogg', 80, 1, -1)
-
-			var/obj/item/bodypart/BP = T.get_bodypart(BODY_ZONE_HEAD)
-			if(BP)
-				BP.receive_damage(36) //so 3 toolbox hits
-
-			T.visible_message("<span class='warning'>[src] curbstomps [T]!</span>", "<span class='warning'>[src] curbstomps you!</span>")
+			if(src.zone_selected == BODY_ZONE_HEAD) //curbstomp specific code
+
+				var/increment = (T.lying/90)-2
+				setDir(increment > 0 ? WEST : EAST)
+				for(var/i in 1 to 5)
+					src.pixel_y += 8-i
+					src.pixel_x -= increment
+					sleep(0.2)
+				for(var/i in 1 to 5)
+					src.pixel_y -= 8-i
+					src.pixel_x -= increment
+					sleep(0.2)
+
+				playsound(src, 'sound/effects/hit_kick.ogg', 80, 1, -1)
+				playsound(src, 'sound/weapons/punch2.ogg', 80, 1, -1)
+
+				var/obj/item/bodypart/BP = T.get_bodypart(BODY_ZONE_HEAD)
+				if(BP)
+					BP.receive_damage(36) //so 3 toolbox hits
+				
+				T.visible_message("<span class='warning'>[src] curbstomps [T]!</span>", "<span class='warning'>[src] curbstomps you!</span>")
+
+				log_combat(src, T, "curbstomped")
+
+			if(src.zone_selected == BODY_ZONE_PRECISE_GROIN) //groinkick specific code
+
+				var/increment = (T.lying/90)-2
+				setDir(increment > 0 ? WEST : EAST)
+				for(var/i in 1 to 5)
+					src.pixel_y += 2-i
+					src.pixel_x -= increment
+					sleep(0.2)
+				for(var/i in 1 to 5)
+					src.pixel_y -= 2-i
+					src.pixel_x -= increment
+					sleep(0.2)
+
+				playsound(src, 'sound/effects/hit_kick.ogg', 80, 1, -1)
+				playsound(src, 'sound/effects/hit_punch.ogg', 80, 1, -1)
+
+				var/obj/item/bodypart/BP = T.get_bodypart(BODY_ZONE_CHEST)
+				if(T.gender == MALE && (ishuman(T)) || (islizard(T)) || (iscatperson(T)))
+					if(BP)
+						BP.receive_damage(25)
+				else
+					if(BP)
+						BP.receive_damage(15)

```

Comment:
```
if(BP && ishuman(T))
    if(T.gender == MALE)
        BP.receive_damage(25)
    else
        BP.receive_damage(15)
```

Reply:
Proc already checks if they're human.

Reply:
These were meant to be species specific as a feature but It would be better balanced if it applied to everyone.

Reply:
Nah, we are not making another cs, something should be different or it's boring.

Reply:
Too late now ive already commited it and it will be fun, but I could try to get the male damageset to force the scream emote